tests/core/mempool/test_mempool_manager.py

```diff
@@ -351,7 +351,7 @@ def make_bundle_spends_map_and_fee(
     spend_bundle: SpendBundle, npc_result: NPCResult
 ) -> Tuple[Dict[bytes32, BundleCoinSpend], uint64]:
     bundle_coin_spends: Dict[bytes32, BundleCoinSpend] = {}
-    eligibility_and_additions: Dict[bytes32, Tuple[bool, List[Coin]]] = {}
+    eligibility_and_additions: Dict[bytes32, Tuple[bool, List[Coin], bool]] = {}
```

Comment:
this `Tuple` might be complex enough to warrant a named type now. I'm especially worried about two unnamed `bool` fields. they can easily be mixed up